multimodal-search-pdf/pods/segmenters.py

```diff
@@ -164,7 +164,7 @@ def segment(self,  uri: str, buffer: bytes, mime_type: str, *args, **kwargs) ->
         """
         chunks=[]
         if uri:
-            if mime_type == 'image/png' or 'image/jpeg':
+            if mime_type == 'image/png' or mime_type == 'image/jpeg':
```

Comment:
if `mime_type` in [''mage/png' , 'image/jpeg']
```suggestion
            if `mime_type` in [''mage/png' , 'image/jpeg']:
```

Reply:
how about changing it to `if mime_type.startswith('image'):`?

Reply:
can be done as well yes